packages/react-native-branch/android/.gitignore

```diff
@@ -0,0 +1,261 @@
+
```

Comment:
do we really need the gitignore file here? please consider remove this or add necessary exclusions into https://github.com/expo/config-plugins/blob/master/.gitignore

Reply:
Yes, there are some files under `android/build` directory.